src/Data/Vector/Linear.hs

```diff
@@ -64,15 +63,15 @@ newtype V (n :: Nat) (a :: *) = V (Vector a)
   -- Vectors at the moment.
 
 theLength :: forall n. KnownNat n => Int
-theLength = fromIntegral (natVal' @n (proxy# @_))
+theLength = fromIntegral (natVal @n Proxy)
```

Comment:
What's the motivation for that? I used proxy to make sure that there was no value at run time.

Reply:
I thought it was clearer at no real performance cost, but I'll revert this change.